modules/openapi-generator-gradle-plugin/bin/main/org/openapitools/generator/gradle/plugin/extensions/OpenApiGeneratorGenerateExtension.kt

```diff
@@ -0,0 +1,286 @@
+/*
+ * Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.openapitools.generator.gradle.plugin.extensions
+
+import org.gradle.api.Project
+import org.gradle.kotlin.dsl.listProperty
+import org.gradle.kotlin.dsl.property
+
+/**
+ * Gradle project level extension object definition for the generate task
+ *
+ * @author Jim Schubert
+ */
+open class OpenApiGeneratorGenerateExtension(project: Project) {
```

Comment:
why is this file part of this PR?

Reply:
no idea. should i revert it? my git blame is telling me it was added 4 months ago by Jim Schubert in #162